update ip whitelist (#1165)
vesoft-inc/nebula-docs: docs-2.0/7.data-security/1.authentication/2.management-user.md

The `root` user with the **GOD** role can run `CREATE USER` to create a new user.

- Syntax

  ```ngql
  CREATE USER [IF NOT EXISTS] <user_name> [WITH PASSWORD '<password>'] [WITH IP WHITELIST <ip_list>];
  ```

  - `IF NOT EXISTS`: Detects if the related user name exists. If it does not exist, a new one will be created.
  - `user_name`: Name of the user to be created.
  - `password`: Password of the user name.

>   - `ip_list`(Enterprise): IP address whitelist. The user can connect to Nebula Graph through the IP listed in the list. Use commas to separate multiple IP addresses.

Reviewer: ```suggestion
  - `ip_list`(Enterprise): Sets the IP address whitelist. The user can connect to Nebula Graph only from IP addresses in the list. Use commas to separate multiple IP addresses.
```

## ALTER USER

> The `root` user with the **GOD** role can run `ALTER USER` to set a new password and IP address whitelist for a user. The old password is not needed when setting a new one.

Reviewer: ```suggestion
The `root` user with the **GOD** role can run `ALTER USER` to set a new password and IP address whitelist for a user. The old password is not needed when altering the user.
```
